twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Header/MemberChannelRow.tsx

```diff
@@ -107,16 +107,17 @@ export default (props: Props) => {
     );
   } else {
     let menu: any = [
-      props.userType === 'pending-email' && {
-        text: (
-          <div>
-            {Languages.t(
-              'scenes.client.channels_bar.modals.parts.channel_member_row.menu.re_send_invitation',
-            )}
-          </div>
-        ),
-        onClick: () => console.log('clicked'),
-      },
+      // Not implemented yet
+      // props.userType === 'pending-email' && {
+      //   text: (
+      //     <div>
+      //       {Languages.t(
+      //         'scenes.client.channels_bar.modals.parts.channel_member_row.menu.re_send_invitation',
+      //       )}
+      //     </div>
+      //   ),
+      //   onClick: () => console.log('clicked'),
+      // },
```

Comment:
We use git, you can stop commenting to find it in some weeks, you can use the git history for that

Reply:
Removed